src/services/form-validation.ts

```diff
@@ -383,6 +389,21 @@ const generateSchemaReducer = (
         }
         break
       }
+      case 'boolean': {
+        if (formElement.type === 'boolean') {
```

Comment:
Wont this always be true? Do we want to keep the log below?

Reply:
mistake, originally had this as part of a larger `case`